src/apps/relay/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+get_filename_component(MZ_SOURCE_DIR ${CMAKE_SOURCE_DIR}/src ABSOLUTE)
+get_filename_component(GENERATED_DIR ${CMAKE_CURRENT_BINARY_DIR}/generated ABSOLUTE)
+
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
```

Comment:
Nit: Should this be somewhere "shared", given we use this logic for vpn aswell?

Reply:
Yes... but I don't know how yet!

Reply:
We could move it to the top-level `CMakeLists.txt` in which case these variables will be set for every target that gets built. It would be a lot cleaner, but it will probably require a bunch of refactoring in how we handle the dummy builds and unit tests (which are a bit of a hack anyways).